### ISSUE TITLE
Why doesn't the counter in data_iter increase?

### ISSUE THREAD

**Issue by nateraw** [codertimo/BERT-pytorch#39]
I am currently playing around with training and testing the model. However, as I implemented the test section, I'm noticing that during the LM training, your counter doesn't increase when looping over data_iter found in [pretrain.py](https://github.com/codertimo/BERT-pytorch/blob/master/bert_pytorch/trainer/pretrain.py). This would cause problems when calculating the average loss/accuracy, wouldn't it? 

![image](https://user-images.githubusercontent.com/32437151/47596857-a5f47680-d946-11e8-9344-89cda1311bcb.png)

**Comment by nateraw** [comment]
Looks like this may be related to #32

**Comment by nateraw** [comment]
alpha0.0.1a4

**Comment by nateraw** [comment]
@codertimo I just read over some more of the comments on #32...I'll update to latest version and then try again. Will report back ASAP

**Comment by codertimo** [comment]
It seems like you are not the using original code. Did you changed something? 
The loss are not looks like 0.0.1a4 version

**Comment by nateraw** [comment]
I had changed the format of the output to dynamically work with some tests I was running. No actual changes to logic were made (but I could be wrong, haha). I'll close this for now and will reopen an issue if I find it continues (on your unaltered code).
